src/main/java/com/tmtravlr/additions/gui/view/components/input/GuiComponentIntegerInput.java

```diff
@@ -202,12 +193,23 @@ public void setFocused(boolean focused) {
 			this.checkLimits();
 		}
 	}
-	
+
+	private NumberFormatException lastException;
+
+	/**
+	 * Note from sschr15: this was edited so you no longer get spammed
+	 * when the color field is empty
+	 */
 	public int getInteger() {
 		try {
-			return Integer.parseInt(this.getText());
+			int theInt = Integer.parseInt(this.getText());
+			lastException = null;
+			return theInt;
```

Comment:
Suggestion: Instead of checking the last exception, check that this.getText() is not empty (so it doesn't even need to throw an exception)
```suggestion
			return this.getText().isEmpty() ? Math.max(0, this.minimum) : Integer.parseInt(this.getText());
```